helm/values.schema.json

```diff
@@ -0,0 +1,1713 @@
+                "examples": [
+                  {
+                    "cpu": "1000m",
+                    "memory": "512Mi"
+                  }
+                ],
+                "properties": {
+                  "cpu": {
+                    "$id": "#/properties/deployment/properties/resources/properties/limits/properties/cpu",
+                    "type": "string",
+                    "pattern": "([0-9]+m|[0-9]+.[0-9]+)",
+                    "title": "The cpu schema",
+                    "description": "CPU limit of Connaisseur container. (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)",
+                    "examples": [
+                      "1000m"
+                    ]
+                  },
+                  "memory": {
+                    "$id": "#/properties/deployment/properties/resources/properties/limits/properties/memory",
+                    "type": "string",
+                    "pattern": "^[0-9]+(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)$",
+                    "title": "The memory schema",
+                    "description": "Memory limit of Connaisseur container. (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory)",
+                    "examples": [
+                      "512Mi"
+                    ]
+                  }
+                },
+                "additionalProperties": true
+              },
+              "requests": {
+                "$id": "#/properties/deployment/properties/resources/properties/requests",
+                "type": "object",
+                "title": "The requests schema",
+                "description": "Resource requests of Connaisseur container. (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#how-pods-with-resource-requests-are-scheduled)",
+                "examples": [
+                  {
+                    "cpu": "100m",
+                    "memory": "128Mi"
+                  }
+                ],
+                "properties": {
+                  "cpu": {
+                    "$id": "#/properties/deployment/properties/resources/properties/requests/properties/cpu",
+                    "type": "string",
+                    "pattern": "([0-9]+m|[0-9]+.[0-9]+)",
+                    "title": "The cpu schema",
+                    "description": "CPU requests of Connaisseur container. (https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu)",
+                    "default": "",
+                    "examples": [
+                      "100m"
+                    ]
+                  },
+                  "memory": {
+                    "$id": "#/properties/deployment/properties/resources/properties/requests/properties/memory",
+                    "type": "string",
+                    "pattern": "^[0-9]+(E|P|T|G|M|k|Ei|Pi|Ti|Gi|Mi|Ki)$",
```

Comment:
In light of that, I'd in favor of removing it. Though maybe for the CPU with only m (or does that also allow k, G, ...?) we can do the validation. Not super sold, so would be fine with validating or not there

Reply:
I'll remove it - also in light of the fact, that Connaisseur installation will fail anyway if there is a weird value because k8s checks that anyway :) Thanks for the hint!